Framework/DataHandling/src/LoadSampleShape.cpp

```diff
@@ -0,0 +1,200 @@
+#include "MantidDataHandling/LoadSampleShape.h"
+
+#include "MantidAPI/IFileLoader.h"
```

Comment:
Just moving the include to the cpp file is not an answer, you should not include an interface that you are not using.